firefox-ios/Client/Frontend/TrackingProtection/CertificatesHandler.swift

```diff
@@ -32,7 +33,9 @@ class CertificatesHandler {
                 let certificate = try Certificate(derEncoded: Array(certificateData))
                 certificates.append(certificate)
             } catch {
-                print("Error decoding certificate: \(error)")
+                DefaultLogger.shared.log("\(error)",
```

Comment:
@dicarobinho In a future PR or task, please use your original `Error decoding certificate: \(error)"` and `"Error fetching data: \(error)"` as the log messages. Otherwise anyone seeing an error won't know where it's coming from. Thanks! 😃